sys/net/gnrc/transport_layer/tcp/Kconfig

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_TCP
+    bool "Configure GNRC_TCP"
+    depends on MODULE_GNRC_TCP
+    help
+        Configure the GNRC_TCP using Kconfig.
+
+if KCONFIG_MODULE_GNRC_TCP
+
+config GNRC_TCP_CONNECTION_TIMEOUT_DURATION
+    int "Timeout duration for user calls in microseconds"
+    default 120000000
+    help
+        Timeout duration for user calls. Default value is 120000000 microseconds
+        (2 minutes).
+
+config GNRC_TCP_MSL
+    int "Maximum segment lifetime (MSL) in microseconds"
+    default 30000000
+    help
+        Maximum segment lifetime (MSL) in microseconds. Default value is 30
+        seconds.
+
+config GNRC_TCP_MSS_MULTIPLICATOR
+    int "Number of MSS sized packets stored in receive buffer"
+    default 1
+    help
+        Configure MSS Multiplicator i.e. number of MSS sized packets stored in
+        receive buffer.
+
+config GNRC_TCP_RCV_BUFFERS
+    int "Number of preallocated receive buffers"
```

Comment:
The description could be better. This value determines how many parallel TCP connection can be active at the same time.

Reply:
Added. Thanks for the comment

Reply:
Its still the same description